src/installer-win/SetPath.ps1

```diff
@@ -0,0 +1,49 @@
+param (
```

Comment:
Would you mind making a note to update the install instructions for Windows when we next have a release? Maybe we can just start another 'running items for 0.2 release' note.

Reply:
Created #424